src/riscv-legacy-integration.adoc

```diff
@@ -220,11 +220,8 @@ When an extended CSR is used with another CSR instruction (<<CSRRWI>>,
 * The final address is updated as specified in <<extended_CSR_writing>> for an
 XLEN write.
 * XLEN bits are read from the capability address field and written to an output
```

Comment:
CLEN bits are read, which is what we're trying to fix

Reply:
Yes, sorry. I inverted the sense of what I wanted to say. Fixed now.